docker/Makefile

```diff
@@ -48,28 +49,39 @@ platformio/Dockerfile.log: platformio/Dockerfile | check-docker
 	| tee -a $@.part
 	mv $@.part $@
 
-.PHONY: flash
-flash:
+.PHONY: pio-flash pio-upload
+pio-flash pio-upload:
```

Comment:
the container dependency and check if docker is installed is missing
`pio-flash pio-upload: | docker-platformio check-docker`